.github/mergify.yml

```diff
@@ -0,0 +1,70 @@
+queue_rules:
+  - name: urgent
+    speculative_checks: 2
+    batch_size: 2
+    conditions:
+      - check-success=CI
+      - check-success=Coverage
+
+  - name: medium
+    speculative_checks: 2
+    batch_size: 3
+    conditions:
+      - check-success=CI
+      - check-success=Coverage
+
+  - name: default
+    speculative_checks: 2
+    batch_size: 4
+    conditions:
+      - check-success=CI
+      - check-success=Coverage
+
+pull_request_rules:
+  - name: automatic update for PR marked as “Ready-to-Go“
+    conditions:
+      - -conflict # skip PRs with conflicts
+      - -draft # filter-out GH draft PRs
+      - label="Ready-to-Go"
```

Comment:
Streamlined Defaults: Can we update PRs by default, without requiring a specific label?
```suggestion
```